### PR TITLE
update flashing page with STM32F7 erasure note and others

### PR DESCRIPTION
**Pull request by nmschulte**
edit the page quite a bit

prefer "update" over "flash"

add `dfu-util` section

add STM32F7 "full erase" notice

### CODE REVIEW

File: docs/Intro-Start-Here/HT-Flashing-The-Firmware.md [hunk 1]
```diff
@@ -4,36 +4,94 @@ sidebar_position: 4
...
+detected. Click *Connect* to connect the ECU to the programmer. Click the *Read* button to read the device memory and to
+validate that the ECU is connected.
+
+Once validated, click the *Open File* button and load the file marked *fome.bin* within the firmware bundle downloaded
+earlier. This is a binary file containing the ECU firmware. Now click the *Download* button to write this firmware to
+the ECU. Give it up to several minutes to download and once the status console confirms successful download, disconnect
+from the programmer then unplug the ECU and plug it back in. The ECU should now connect to the computer and TS normally
+again.
+
+## dfu-util
```
Comment (nmschulte) [inline docs/Intro-Start-Here/HT-Flashing-The-Firmware.md:81]: should be one level deeper

File: docs/Intro-Start-Here/HT-Flashing-The-Firmware.md [hunk 2]
```diff
@@ -4,36 +4,100 @@ sidebar_position: 4
...
 
-If this does not complete successfully, the firmware may need to be loaded using STM32 Cube Programmer covered later.
+If the flashing does not complete successfully, the firmware may need to be loaded using another mechanism documented
+below in [Other firmware flashing tools](#other-firmware-flashing-tools).
 
 ## Setting up TunerStudio
 
-Open TS with the ECU plugged in, it should recognise that the ECU definition is out of date and come up with a prompt to download the new definition.
+With the ECU connected, open TunerStudio with the associated project.  TunerStudio will recognise that the project's ECU
+definition is out of date and display a prompt to download or supply the new definition.
```
Comment (mck1117) [inline docs/Intro-Start-Here/HT-Flashing-The-Firmware.md:36]: a screenshot here with which button to push would be helpful, it's not entirely clear the first time you see that window

Comment (nmschulte) [reply docs/Intro-Start-Here/HT-Flashing-The-Firmware.md]: I'll add it if you can provide one; I'm not setup for it at the moment.

File: docs/Intro-Start-Here/Updating-FW/popularvehicles.png
Comment (nmschulte) [inline docs/Intro-Start-Here/Updating-FW/popularvehicles.png:1]: @mck1117 this image also says "rusEFI" vs "FOME" -- unsure if the TS bits still reflect this or just outdated iamge

File: docs/Intro-Start-Here/HT-Updating-The-Firmware.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+situations, the FOME console cannot communicate with the firmware and so is unable to configure it into bootloader (or
+DFU) mode for updating.  To flash the firmware, the ECU must be manually entered into the bootloader/DFU mode.
+
+### STM32CubeProgrammer
+
+STM32CubeProgrammer is an officially STMicroelectronics supported tool to flash STM32 processors.  Information about
+and how to download and install the tool can be found on [the STM
+wiki](https://wiki.st.com/stm32mpu/wiki/STM32CubeProgrammer).
+
+For the ECU, you will need to access buttons or pins on the main board to force the processor into bootloader/DFU mode.
```
Comment (nmschulte) [inline docs/Intro-Start-Here/HT-Updating-The-Firmware.md:70]: This whole paragraph could be bumped up a heading, as it applies generally regardless of the tool in use.

**Comment by nmschulte** [comment]
page has been updated invaliding these changes; see new PR: #112

**Pull request closed without merging**